environment.yml

```diff
@@ -1,15 +1,14 @@
-name: binder-env
+name: giddy
 channels:
   - conda-forge
 dependencies:
-  - python=3.11
+  - python
+  - jupyterlab
+  - esda
+  - libpysal
+  - quantecon
+  - scipy
   - geopandas
-  - libspatialindex
-  - matplotlib
-  - rtree
-  - seaborn
   - splot
-  - watermark
   - pip
-  - pip:
-    - git+https://github.com/pysal/giddy.git@main
```

Comment:
Don't we need to keep this in here to install the current `main` of `giddy`?

Reply:
personally, i just use the environment.yml to install depepndencies, so i can have an editable install locally (e.g. `pip install . -e`)

so these changes look fine do me. But i also think james is probably right and we need setuptools_scm in the conda-forge recipe here https://github.com/conda-forge/giddy-feedstock/blob/main/recipe/meta.yaml

Reply:
I am not sure whether we need `giddy/main`. I was mimicking [`libpysal`'s environment file](https://github.com/pysal/libpysal/blob/main/environment.yml). I will defer to you guys.